src/applications/personalization/dashboard-2/components/IconCTALink.jsx

```diff
@@ -26,18 +26,22 @@ const IconCTALink = ({
       onClick={onClick || undefined}
       className={linkClass}
     >
-      <span className="fa-stack fa-sm vads-u-margin-right--1">
-        <i
-          aria-hidden="true"
-          className="fas fa-circle fa-stack-2x vads-u-color--primary-alt-lightest"
-        />
-        <i aria-hidden="true" className={`fas fa-${icon} fa-stack-1x`} />
+      <span className="vads-u-display--flex">
```

Comment:
I needed to rework the layout of this component so that the next would stack as desired on mobile. Looks much better now! 👍